multi_import/importer.py

```diff
@@ -178,42 +195,48 @@ def get_import_queryset(self):
 
     def export(self, empty=False, context=None):
         serializer_context = self.get_export_serializer_context(context)
-        serializer = self.serializer(context=serializer_context)
-        dataset = Dataset(headers=self.get_export_header(serializer))
+        serializers = [
+            serializer_class(context=serializer_context)
+            for serializer_class in self.get_serializer_classes()
+        ]
+        dataset = Dataset(headers=self.get_export_header(serializers))
 
         if not empty:
             for instance in self.get_export_queryset():
-                dataset.append(self.get_export_row(serializer, instance))
+                dataset.append(self.get_export_row(serializers, instance))
 
         return ExportResult(
             dataset=dataset,
             empty=empty,
-            example_row=self.get_example_row(serializer),
+            example_row=self.get_example_row(serializers),
             file_formats=self.file_formats,
             filename=self.get_export_filename(),
         )
 
-    def get_export_header(self, serializer):
+    def get_export_header(self, serializers):
         return [
             field_name
+            for serializer in serializers
             for field_name, field in serializer.get_fields().items()
             if not field.write_only
         ]
 
-    def get_example_row(self, serializer):
+    def get_example_row(self, serializers):
         results = []
-        for _column_name in serializer.fields:
-            results.append("")
+        for serializer in serializers:
+            for _column_name in serializer.fields:
+                results.append("")
         return results
 
-    def get_export_row(self, serializer, instance):
+    def get_export_row(self, serializers, instance):
         results = []
-        representation = serializer.to_representation(instance=instance)
-        for column_name, value in representation.items():
-            field = serializer.fields[column_name]
-            val = fields.to_string_representation(field, value)
-            # TODO: Excel escaping should be done for Excel/CSV formats
-            results.append(strings.excel_escape(val))
+        for serializer in serializers:
+            representation = serializer.to_representation(instance=instance)
+            for column_name, value in representation.items():
+                field = serializer.fields[column_name]
+                val = fields.to_string_representation(field, value)
+                # TODO: Excel escaping should be done for Excel/CSV formats
```

Comment:
Is this still TODO? Line below looks like excel escape?

Reply:
Yeah, I think you're probably right!
This has been here for 4 years, so I was wary of removing the TODO. Removed now.